package.json

```diff
@@ -101,5 +101,7 @@
     "typescript": "4.0.5",
     "url-join": "^4.0.1"
   },
-  "dependencies": {}
+  "dependencies": {
+    "@types/aria-query": "^4.2.1"
+  }
```

Comment:
This seems unrelated to your PR, no?

Reply:
Indeed, though as noted previously I sometimes see transpilation fail when I don't have the aria types installed